### PR TITLE
DHFPROD-6119: Ant Design components are fixed to body element

### PR DESCRIPTION
**Pull request by xnikhil08**
### Description
This is just fixing implementation detail for ant design components, to fix the issue where some components like tooltip, popovers etc., were being fixed to the body element. No tests are required here.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [N/A] Added Tests
  

- ##### Reviewer:

- [N/A] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/date-facet/date-facet.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ const DateFacet: React.FC<Props> = (props) => {
                 onChange={onChange}
                 value={datePickerValue}
                 key={props.name}
+                getCalendarContainer={() => document.getElementById('sideBarContainer') || document.body}
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/components/date-facet/date-facet.tsx:73]: The getCalendarContainer() property is specific to the date / time picker component. For all other components its not needed and getPopupContainer should be used.

**Review by timur-isangulov — APPROVED** [review]

**Review by wooldridge — CHANGES_REQUESTED** [review]
@xnikhil08, nice work! I'm seeing a few issues, not sure if all are related to your changes...

**Explore View**

- Created On menu appears disconnected
- Some tooltips are italic, unlike other tooltips (Close icon tooltip is italic in all views)

https://drive.google.com/file/d/1D7rTSbZ3CDVgXJP0FcQ2rSrCL96i9fC2/view

**Curate View**

- In Mapping details, function menu is throwing an error. (Can you reproduce?)

https://drive.google.com/file/d/1KEVQy_AP93Vf7azO0V_rJcKmbjEU4yHd/view

**Review by brucean52 — CHANGES_REQUESTED** [review]
In Model tile, Tooltip alignment is off the adding structured types.

Modeling-structured

**Comment by xnikhil08** [comment]
> In Model tile, Tooltip alignment is off for adding structured type properties.
> 
> Modeling-structured

@brucean52 - Fixed it but the tooltip will be like the screenshot below.   

image

Discussed with Allison and she agreed that it looks ok.

**Comment by xnikhil08** [comment]
> @xnikhil08, nice work! I'm seeing a few issues, not sure if all are related to your changes...
> 
> **Explore View**
> 
> * Created On menu appears disconnected
> * Some tooltips are italic, unlike other tooltips (Close icon tooltip is italic in all views)
> 
> https://drive.google.com/file/d/1D7rTSbZ3CDVgXJP0FcQ2rSrCL96i9fC2/view
> 
> **Curate View**
> 
> * In Mapping details, function menu is throwing an error. (Can you reproduce?)
> 
> https://drive.google.com/file/d/1KEVQy_AP93Vf7azO0V_rJcKmbjEU4yHd/view

@wooldridge - All the issues are fixed. But i can't reproduce the function dropdown issue that you mentioned. Looks like its not related to this PR.

**Comment by wooldridge** [comment]
@xnikhil08, I'm seeing the Mapping details Name search popup not being fixed to the tile container:

https://drive.google.com/file/d/1LEPlaTJN6UEBI4lltsLUNPiuWAr7kPjS/view

Also, I'm still seeing a lot of the tooltips in italics:

Screen Shot 2020-10-21 at 9 15 13 AM

**Review by wooldridge — CHANGES_REQUESTED** [review]
See my comment.

**Comment by xnikhil08** [comment]
> See my comment.

@wooldridge - Fixed those issues.

**Review by timur-isangulov — CHANGES_REQUESTED** [review]
Hi @xnikhil08, the export and column selector icon tooltips are off. 
Screen Shot 2020-10-21 at 1 24 26 PM
Screen Shot 2020-10-21 at 1 24 31 PM

**Comment by timur-isangulov** [comment]
The tooltips for "Instance" and "JSON" views are bold. 
Screen Shot 2020-10-21 at 1 38 49 PM

**Comment by xnikhil08** [comment]
Opening a separate PR for develop to avoid confusion.

**Pull request closed without merging**